k3s.pkr.hcl

```diff
@@ -2,6 +2,15 @@ variable "k3s_version" {
   type    = string
 }
 
+packer {
+  required_plugins {
+    googlecompute = {
+      source  = "github.com/hashicorp/googlecompute"
+      version = "~> 1"
+    }
+  }
+}
+
 source "googlecompute" "k3s" {
     project_id   = "instruqt"
```

Comment:
should we maybe insert these through variables? that way we can build images in multiple projects and regions (especially with multi-region functionality)